index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Potfolio page</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+    href="https://font.googleapis.com/css2?family=Poppin"rel="stylesheet"/>
+    <link
+    href="https://font.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap"rel="stylesheet"
+    />
+</head>
+<body>
+    <header class="header">
+        <nav class="header_nav">
+            <div class="header_logo"><a class="blue" href="#">My logo</a></div>
+            <div class="header_menu">
```

Comment:
- [ ] Good Job @lopezivan1216, For the logo please make sure you apply the right font and size. As you can see from the screenshot your design is a bit off the design. It's A great skill to be able to follow the original design. I believe you agree :smile: 
## Figma Logo
![figma font](https://user-images.githubusercontent.com/69077061/199953602-b48ff8ed-54a3-49cb-9535-ed061fe88781.png)

## Your Logo
![your font](https://user-images.githubusercontent.com/69077061/199953610-9fa08f98-43c2-4f16-988a-d80eaa2884a4.png)